### PR TITLE
ref: Remove Kafka pipeline consumer, batching-kafka-consumer

### PR DESCRIPTION
**Pull request by tkaemming**
This is all debris from an abandoned experiment and is unsupported.

See SNS-168

### CODE REVIEW

File: src/sentry/runner/commands/consumer.py [hunk 1]
```diff
@@ -1,49 +0,0 @@
-from __future__ import absolute_import, print_function
```
Comment (fpacifici) [inline src/sentry/runner/commands/consumer.py]: I think you need to remove this reference as well:

https://github.com/getsentry/sentry/blob/master/src/sentry/runner/__init__.py#L52

**Review by fpacifici — CHANGES_REQUESTED** [review]

File: src/sentry/coreapi.py [hunk 2]
```diff
@@ -176,25 +174,10 @@ def insert_data_to_database(self, data, start_time=None,
         if attachments is not None:
             attachment_cache.set(cache_key, attachments, cache_timeout)
 
-        # NOTE: Project is bound to the context in most cases in production, which
-        # is enough for us to do `projects:kafka-ingest` testing.
-        project = self.context and self.context.project
-
-        if project and features.has('projects:kafka-ingest', project=project):
```
[the remark this answers is not in the document]
Comment (tkaemming) [reply src/sentry/coreapi.py]: Will do to be extra safe, though I would be shocked if anything does

File: src/sentry/runner/commands/consumer.py (hunk 1 above)
Comment (tkaemming) [reply src/sentry/runner/commands/consumer.py]: That happened with fccd43bff8065a08faa12824fb63b70ef34423aa

Comment (fpacifici) [reply src/sentry/runner/commands/consumer.py]: sorry, I think I was looking to one commit only.

**Review by fpacifici — APPROVED** [review]

File: src/sentry/runner/commands/consumer.py (hunk 1 above)
Comment (tkaemming) [reply src/sentry/runner/commands/consumer.py]: No worries

File: src/sentry/coreapi.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tkaemming) [reply src/sentry/coreapi.py]: I needed to remove some references in the internal repository as well where the feature handler was defined, thanks for leading me down that path

**Pull request merged**